### PR TITLE
Update interactions with Param types across ADRIA more generally

### PR DESCRIPTION
**Pull request by ConnectedSystems**
As in title.

This PR is part of #636 and should be merged in **third**.

### CODE REVIEW

File: ext/AvizExt/AvizExt.jl [hunk 1]
```diff
@@ -523,7 +523,7 @@ function ADRIA.viz.explore(rs::ResultSet)
 
     # Trigger update only after some time since last interaction
     # TODO: Add update notification (spinner animation or something...)
-    up_timer = Timer(x -> x, 0.25)
+    # up_timer = Timer(x -> x, 0.25)
```
Comment (Zapiano) [inline ext/AvizExt/AvizExt.jl:526]: Are you planning to leave this here?

File: src/ADRIA.jl [hunk 2]
```diff
@@ -167,34 +168,34 @@ if ccall(:jl_generating_output, Cint, ()) == 1
 end
 
 
-@setup_workload begin
-    # Putting some things in `setup` can reduce the size of the
-    # precompile file and potentially make loading faster.
-    # ADRIA_DIR = pkgdir(ADRIA)
-    # EXAMPLE_DOMAIN_PATH = joinpath(ADRIA_DIR, "examples", "Example_domain")
+# @setup_workload begin
```
Comment (Zapiano) [inline src/ADRIA.jl:171]: Why dont we need this anymore?

**Review by Zapiano — COMMENTED** [review]
I think it would be easier to review this one after a rebase.

**Comment by ConnectedSystems** [comment]
> I think it would be easier to review this one after a rebase.

Yes, this is part of a series. The rebase happens after #638 is merged in.

**Review by Zapiano — APPROVED** [review]
🚀

**Pull request merged**